common-api-v2/gadi/packages.yaml

```diff
@@ -0,0 +1,714 @@
+      # $ module show python3-as-python
+      # -------------------------------------------------------------------
+      # /apps/Modules/modulefiles/python3-as-python:
+      #
+      # conflict        python2-as-python
+      # setenv          NCI_PYTHONX_AS_PYTHON python3
+      # -------------------------------------------------------------------
+      extra_attributes:
+        environment:
+          set:
+            NCI_PYTHONX_AS_PYTHON: python3
+    buildable: false
+  gcc:
+    externals:
+    - spec: gcc@15.1.0 languages:='c,c++,fortran'
+      prefix: /apps/gcc/15.1.0/wrappers
+      modules: [gcc/15.1.0]
+      extra_attributes:
+        compilers:
+          c: /apps/gcc/15.1.0/wrappers/gcc
+          cxx: /apps/gcc/15.1.0/wrappers/g++
+          fortran: /apps/gcc/15.1.0/wrappers/gfortran
+    - spec: gcc@14.2.0 languages:='c,c++,fortran'
+      prefix: /apps/gcc/14.2.0/wrappers
+      modules: [gcc/14.2.0]
+      extra_attributes:
+        compilers:
+          c: /apps/gcc/14.2.0/wrappers/gcc
+          cxx: /apps/gcc/14.2.0/wrappers/g++
+          fortran: /apps/gcc/14.2.0/wrappers/gfortran
+    - spec: gcc@14.1.0 languages:='c,c++,fortran'
+      prefix: /apps/gcc/14.1.0/wrappers
+      modules: [gcc/14.1.0]
+      extra_attributes:
+        compilers:
+          c: /apps/gcc/14.1.0/wrappers/gcc
+          cxx: /apps/gcc/14.1.0/wrappers/g++
+          fortran: /apps/gcc/14.1.0/wrappers/gfortran
+    - spec: gcc@13.2.0 languages:='c,c++,fortran'
+      prefix: /apps/gcc/13.2.0/wrappers
+      modules: [gcc/13.2.0]
+      extra_attributes:
+        compilers:
+          c: /apps/gcc/13.2.0/wrappers/gcc
+          cxx: /apps/gcc/13.2.0/wrappers/g++
+          fortran: /apps/gcc/13.2.0/wrappers/gfortran
+    - spec: gcc@12.2.0 languages:='c,c++,fortran'
+      prefix: /apps/gcc/12.2.0/wrappers
+      modules: [gcc/12.2.0]
+      extra_attributes:
+        compilers:
+          c: /apps/gcc/12.2.0/wrappers/gcc
+          cxx: /apps/gcc/12.2.0/wrappers/g++
+          fortran: /apps/gcc/12.2.0/wrappers/gfortran
+    - spec: gcc@11.1.0 languages:='c,c++,fortran'
+      prefix: /apps/gcc/11.1.0/wrappers
+      modules: [gcc/11.1.0]
+      extra_attributes:
+        compilers:
+          c: /apps/gcc/11.1.0/wrappers/gcc
+          cxx: /apps/gcc/11.1.0/wrappers/g++
+          fortran: /apps/gcc/11.1.0/wrappers/gfortran
+    - spec: gcc@10.3.0 languages:='c,c++,fortran'
+      prefix: /apps/gcc/10.3.0/wrappers
+      modules: [gcc/10.3.0]
+      extra_attributes:
+        compilers:
+          c: /apps/gcc/10.3.0/wrappers/gcc
+          cxx: /apps/gcc/10.3.0/wrappers/g++
+          fortran: /apps/gcc/10.3.0/wrappers/gfortran
+    - spec: gcc@8.5.0 languages:='c,c++,fortran'
+      prefix: /usr
+      extra_attributes:
+        compilers:
+          c: /usr/bin/gcc
+          cxx: /usr/bin/g++
+          fortran: /usr/bin/gfortran
+  intel-oneapi-compilers:
+    externals:
+    - spec: intel-oneapi-compilers@2025.2.0
```

Comment:
The intel compilers don't seem to have the `languages:='c,c++,fortran'` part of the `spec` which tells `spack` it is a compiler.